gems/pending/spec/spec_helper.rb

```diff
@@ -1,4 +1,12 @@
 require_relative '../bundler_setup'
+
+require 'coveralls'
+Coveralls.wear! do
+  add_filter("/spec/")
+  add_filter("/test/")
+  add_filter("test.rb")
```

Comment:
We have a test.rb?  Also, we should probably ignore any test directories buried in the subdirs...e.g. VMwareWebService/test